### ISSUE TITLE
Property 'client' does not exist on type 'Process'

### ISSUE THREAD

**Issue by academici** [universal-vue/uvue#30]
**Describe the bug**
```bash
ERROR in /home/vostrikov/site/src/registerServiceWorker.ts

5:54 Property 'client' does not exist on type 'Process'.
    3 | import { register } from "register-service-worker";
    4 | 
  > 5 | if (process.env.NODE_ENV === "production" && process.client) {
      |                                                      ^
    6 |   register(`${process.env.BASE_URL}service-worker.js`, {
    7 |     ready() {
    8 |       console.log(
```

**To Reproduce**
Create vue cli project with all features, sass (dart) preprocessor
Add ssr plugin
Run ssr:fix and ssr:fix-vuex
Run ssr:build

**Additional context**

Please indicate versions of:
```
node: 11.12.0
vue-cli: 3.5.1
uvue: v0.1.0-alpha.18
os: ubuntu 18.04
browser: chrome 72
```

**Comment by academici** [comment]
When server is running get an error:
```json
{"level":50,"time":1552736025593,"msg":"navigator is not defined","pid":27483,"hostname":"dmitry-desktop","type":"Error","stack":"ReferenceError: navigator is not defined\n    at ce (server-bundle.js:1:14947)\n    at Module.de6b (server-bundle.js:1:15910)\n    at n (server-bundle.js:1:196)\n    at Object.0 (server-bundle.js:1:1318)\n    at n (server-bundle.js:1:196)\n    at module.exports.0 (server-bundle.js:1:1279)\n    at Object.<anonymous> (server-bundle.js:1:1288)\n    at o (/home/vostrikov/site/node_modules/vue-server-renderer/build.prod.js:1:77379)\n    at /home/vostrikov/site/node_modules/vue-server-renderer/build.prod.js:1:77972\n    at new Promise (<anonymous>)","v":1}
```

**Comment by chymz** [comment]
Can you test this, as workaround: 

Create a file `src/types.d.ts`, with this code:

```ts
import '@uvue/core';
```

**Comment by chymz** [comment]
You can do this too:

In `tsconfig.json`:

```json
{
  "compilerOptions": {
    "types": ["webpack-env", "@uvue/core"],
  }
}
```

**Comment by chymz** [comment]
Fixed in `0.1.0-alpha.19`
